src/components/ButtonPanel.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const ButtonPanel = () => (
+  <div>
+    <div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button">÷</button>
+    </div>
+    <div>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button">X</button>
+    </div>
+    <div>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button">-</button>
```

Comment:
These need to be `<Button />` components actually, not `HTML` "button" types
[ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png) ]